package.json

```diff
@@ -28,6 +28,10 @@
     "registry": "https://registry.npmjs.org/",
     "access": "public"
   },
+  "engines": {
+    "node": "6.14.4"
```

Comment:
I read that this is to fix a 502 error but can you try with node 8.15.0? do you have a link to where the error was documented?

Reply:
Have made the changes you requested. No do not have the link, just read in one of the issues